packages/frontend/src/config/envParsers.js

```diff
@@ -0,0 +1,19 @@
+export const envValIsSet = (envVal) => typeof envVal === "string";
+
+export const parseBooleanFromShell = (envVal) =>
+    envValIsSet(envVal) ? envVal === "yes" || envVal === "true" : undefined;
+
+export const parseObjectFromShell = (envVal) => {
+    if (envValIsSet(envVal)) {
```

Comment:
Since you're already catching the invalid JSON parse in the `try`/`catch` you can omit this check go straight to the `JSON.parse`. Makes it a little more succinct since the return value is the same whether the variable is unset or invalid JSON.